.git-authors

```diff
@@ -8,14 +8,15 @@ pairs:
   nw: Nathan Wang; nathan.wang2
   sc: Serena Cao; wcao0630
   mw: Matt Wills; mattwills8
+  ag: Arun Giridharan; linkin-park
```

Comment:
Hmm what is `linkin-park`? If you are using gmail, this should be the first part

Reply:
okay :) 
my bad thought need to add username of github after ;